tests/unit/views/pacbio/PacbioLibraryBatchCreate.spec.js

```diff
@@ -53,11 +31,12 @@ describe('PacbioLibraryBatchCreate.vue', () => {
         }
       },
     ]
-    const { wrapperObj, createStoreObj } = mountWithStore({
+    const { wrapperObj, storeObj } = mountWithStore(PacbioLibraryBatchCreate, {
```

Comment:
You can do something like

`const { wrapperObj, storeObj: pacbioLibraryBatchCreateStore } = mountWithStore(PacbioLibraryBatchCreate, { ...`

or it might be the other way round.